packages/dom/src/serialize-frames.js

```diff
@@ -9,7 +9,7 @@ function setBaseURI(dom) {
   let $base = document.createElement('base');
   $base.href = dom.baseURI;
 
-  dom.querySelector('head').prepend($base);
+  dom.querySelector('head')?.prepend($base);
```

Comment:
instead of skipping this we can append on `dom.documentElement` equivalent right ?

Reply:
Comment above says it has to be added on <head> tag.

Reply:
Oh, got it fair enough.